examples/pytorch/speech-recognition/run_speech_recognition_ctc.py

```diff
@@ -579,12 +632,24 @@ def remove_special_characters(batch):
         cache_dir=model_args.cache_dir,
         config=config,
         use_auth_token=data_args.use_auth_token,
+        ignore_mismatched_sizes=True,
     )
 
     # freeze encoder
     if model_args.freeze_feature_encoder:
         model.freeze_feature_encoder()
 
+    # if attn adapter is defined, freeze all non-adapter weights
+    if model.config.adapter_attn_dim is not None:
+        model.init_adapter_layers()
+        # first we freeze the whole base model
+        model.freeze_base_model()
+
+        # next we unfreeze all adapter layers
```

Comment:
Do we need to unfreeze the adapter weights? They don't get frozen in `model.freeze_base_model()`

Reply:
They do get frozen in `model.freeze_base_model()` (adapter attention weights are part of it)